pkg/platform/model/vcs.go

```diff
@@ -218,6 +218,24 @@ func LanguageFromNamespace(ns string) string {
 	return values[1]
 }
 
+func IsCustomNamespace(ns string) bool {
+	supportedNamespaces := []NamespaceType{
+		NamespacePackage,
+		NamespaceBundle,
+		NamespaceLanguage,
+		NamespacePlatform,
+		NamespaceOrg,
```

Comment:
Org is a custom namespace we would want to surface.

Also this logic feels veeery specific to `state manifest`, suggest moving the code there.